.vscode/settings.json

```diff
@@ -7,8 +7,24 @@
   "editor.rulers": [100],
   "editor.tabSize": 2,
 
-  "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
+  // Inform VSCode that it should use the version of TypeScript that is installed under this project
```

Comment:
is it ok to have comments in json? it lights up red in the github pr review ui

Reply:
the file in question is JSONC, not JSON.
i'll add something to the PR that fixes the visual bug, one moment

Reply:
see https://github.com/t3-oss/create-t3-app/pull/1181/commits/aa558135fe6d87e93d24917ce50fe9903f659d36

Reply:
> I checked out the pr and installed packages, then broke an [eslint rule that's included in the recommendations](https://ota-meshi.github.io/eslint-plugin-astro/rules/no-conflict-set-directives/), but did not get an error in VSCode.

I get the expected error just fine:

![image](https://user-images.githubusercontent.com/5511220/218266097-bff65268-d1fc-4c4f-9779-f874e79db5db.png)

try recloning the repo from scratch maybe and ensure that you have all of the proper extensions installed?

Reply:
another thing to try is for any personal settings.json settings that conflict with ones in the workspace settings.json